
From: Ronald Bultje <rbultje@ronald.bitfreak.net>

attached patch correctly makes the zr36067 driver depend on i2c-ago-bit in
the kernel config.  Bug reported and patch sent to me by Adrian Bunk
<bunk@fs.tum.de> (6/21).  It wasn't signed off.

Signed-off-by: Ronald Bultje <rbultje@ronald.bitfreak.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/media/video/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/media/video/Kconfig~zr36067-driver-correct-i2c-algo-bit-dependency-in-kconfig drivers/media/video/Kconfig
--- 25/drivers/media/video/Kconfig~zr36067-driver-correct-i2c-algo-bit-dependency-in-kconfig	2004-08-30 00:03:44.000780432 -0700
+++ 25-akpm/drivers/media/video/Kconfig	2004-08-30 00:03:44.003779976 -0700
@@ -155,7 +155,7 @@ config VIDEO_STRADIS
 
 config VIDEO_ZORAN
 	tristate "Zoran ZR36057/36067 Video For Linux"
-	depends on VIDEO_DEV && PCI && I2C
+	depends on VIDEO_DEV && PCI && I2C_ALGOBIT
 	help
 	  Say Y for support for MJPEG capture cards based on the Zoran
 	  36057/36067 PCI controller chipset. This includes the Iomega
_
